modules/ROOT/pages/Development/BeginnersGuide/project_setup.adoc

```diff
@@ -99,14 +118,20 @@ image:BeginnersGuide/simpleMod/Wwise_integrate.gif[image]
 
 == Generate Visual Studio Files
 
-Next, you'll need to generate the remaining files for the project with Unreal Engine's "Generate Visual Studio Project Files" tool. 
+Next, you'll need to generate the remaining files for the project
+with Unreal Engine's "Generate Visual Studio Project Files" tool. 
+
+To do this, you'll need to run a command,
+unless you already have another version of Unreal Engine installed.
+If you do, right-click on the `.uproject` file and select `Generate Visual Studio project files`.
+Go ahead and skip down to the link:#_project_compilation[next section] if this works for you.
 
-To do this, you'll need to run a command, unless you already have another version of Unreal Engine installed. If you do, right-click on the `.uproject` file and select `Generate Visual Studio project files`.  Go ahead and skip down to the xref:#_project_compilation[next section] if this works for you.
+Otherwise, you'll need to run a tool from the command line to generate the project files.
 
-Otherwise, you'll need to run a tool from the command line to generate the project files. The command will follow the form below:
+The command will follow the form below:
 
 ```
-"<path to your Unreal build tool here (it installed with the custom Editor)>" -projectfiles -project="<path to your .uproject file here>" -game -rocket -progress
+"<path to your Unreal build tool here (it was installed with the custom Editor)>" -projectfiles -project="<path to your .uproject file here>" -game -rocket -progress
 ```
 
 Here is an example of a complete command.
```

Comment:
Should we modify the path below to have `SatisfactoryModLoader` instead of `SMLExampleMod`? Goes in line with the fact we use one project for multiple mods as well as the name of the folder when cloning the repo

Reply:
(Had to do this in another commit since the line wasn't in a segment close enough to changed ones to comment on it)